packages/toolkit/form-field/form-field-wrapper.component.scss

```
@@ -868,106 +1055,113 @@
-    color: var(--_color-text);
-  }
+  grid-area: label;
+  position: relative;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  padding-inline-start: 0;
+  margin: 0;
+  min-block-size: var(--_selection-row-min-block-size);
+  padding-block: var(--_selection-row-padding-block);
+  padding-inline: var(--_selection-row-padding-inline) 0;
```

Stylelint (17.6.0)

[error] Unexpected shorthand "padding-inline" after "padding-inline-start" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)